src/tabs/default.scss

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import './tabs';
+
+@mixin iui-tabs-default {
+  @include iui-tabs;
+
+  > li {
+    > button {
+      height: $iui-baseline * 3;
+      padding: 0 $iui-m;
+      box-sizing: border-box;
+      @include themed {
+        background-color: t(iui-color-background-2);
+        border: 1px solid t(iui-color-background-5);
+      }
+
+      .iui-label {
+        text-align: left;
+
+        .iui-description {
```

Comment:
These classnames are too generic

Reply:
I'm using the same class names / structure from within the header component.  Should I use `.iui-tab-label` and `.iui-tab-description` instead?  I'm assuming if yes, we should eventually (in a separate PR) make the same change within the header?

Reply:
Yeah, I would say so. The `.iui-label` class is especially bad here, because a user might have a reason to place, say, an `.iui-button` (which includes its own `.iui-label`) inside tab.

Also, longer names would reduce specificity.